frappe/public/js/frappe/file_uploader/WebLink.vue

```diff
@@ -13,6 +13,14 @@
 				v-model="url"
 			>
 		</div>
+		<div class="input-group">
+			<input
+				type="text"
+				class="form-control"
+				:placeholder="__('URL Title')"
```

Comment:
`Document Name` instead?
```suggestion
				:placeholder="__('Document Name')"
```

Reply:
@surajshetty3416 I thought of that but the could be a url to an web page so I went with the existing convention
If you still suggest Document Name. I see no harm

Reply:
"Document Name" is confusing. Maybe refer to it as a File - "File Name"?

Reply:
Updated @surajshetty3416